errors/arsenalErrors.json

```diff
@@ -476,6 +476,10 @@
     "code": 409,
     "description": "The bucket you tried to delete has an ongoing multipart upload."
   },
+  "LocationNotFound": {
+    "code": 424,
+    "description": "the object data location does not exist"
```

Comment:
Just nitpicking..Uppercase and period!

Reply:
Right, did not pay enough attention to the format of the other messages!